modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIPropertyRelevantObjectEntryModelListener.java

```diff
@@ -96,7 +103,36 @@ private void _scheduleAPIProperties(long apiPropertyId) {
 			});
 	}
 
-	private boolean _validate(
+	private void _validate(
+		long apiSchemaId, String objectFieldERC,
+		String objectRelationshipNames) {
+
+		try {
+			if (!_objectEntryHelper.isValidObjectEntry(
+					apiSchemaId, "L_API_SCHEMA")) {
+
+				throw new ObjectEntryValuesException.InvalidObjectField(
+					null, "An API property must be related to an API schema",
+					"an-api-property-must-be-related-to-an-api-schema");
+			}
+
+			if (!_validateAPIPropertyFields(
```

Comment:
I think the name of the method is not following the pattern. You have a method `private void _validate(....)` and another one `private boolean _validateAPIPropertyFields(...`

What about `_isValidAPIPropertyFields(....)` ??

Reply:
You are right! We must follow the pattern depending on if it is valid or the case we are validating things